migrations.md

```diff
@@ -0,0 +1,11 @@
+If a contributor wants to change the schema, they must create a migration. Current migrations have been created with the following commands:
```

Comment:
There's a lot of instruction in this file that is not specifically concerned with migrations.  I would prefer to make this more concise, and place it in the `CONTRIBUTING.md` file, similar to the floworky approach.